src/main/java/org/squiddev/plethora/integration/roost/IntegrationRoost.java

```diff
@@ -0,0 +1,158 @@
+package org.squiddev.plethora.integration.roost;
+
+import com.google.common.collect.Maps;
+import com.timwoodcreates.roost.Roost;
+import com.timwoodcreates.roost.data.DataChicken;
+import com.timwoodcreates.roost.data.DataChickenModded;
+import com.timwoodcreates.roost.data.DataChickenVanilla;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import org.squiddev.plethora.api.Injects;
+import org.squiddev.plethora.api.converter.ConstantConverter;
+import org.squiddev.plethora.api.converter.DynamicConverter;
+import org.squiddev.plethora.api.meta.IMetaProvider;
+import org.squiddev.plethora.api.meta.ItemStackContextMetaProvider;
+import org.squiddev.plethora.api.meta.NamespacedMetaProvider;
+import org.squiddev.plethora.api.method.IPartialContext;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@Injects(Roost.MODID)
+public final class IntegrationRoost {
+	private IntegrationRoost() {
+	}
+
+	//TODO Define a meta-provider for Vanilla's EntityChicken
```

Comment:
As you've got an `Entity` → `DataChicken` converter, `META_DATA_CHICKEN` should show up fine.

Reply:
Yeah, I think that may have been an earlier to-do item that I just hadn't cleaned out...